libcxx/test/support/nasty_string.h

```diff
@@ -16,6 +16,7 @@
 
 #include "make_string.h"
 #include "test_macros.h"
+#include "constexpr_char_traits.h" // is_nonoverlapping_range
```

Comment:
```suggestion
#include "constexpr_char_traits.h" // is_overlapping_range
```

Reply:
Applied with the new name `is_pointer_in_range`.